src/node/build/bundle.ts

```diff
@@ -15,14 +15,24 @@ import { sanitizeFileName, slash } from '../shared'
 import { task } from '../utils/task'
 import { buildMPAClient } from './buildMPAClient'
 
-// A list of default theme components that should only be loaded on demand.
-const lazyDefaultThemeComponentsRE =
-  /VP(HomeSponsors|DocAsideSponsors|TeamPage|TeamMembers|LocalSearchBox|AlgoliaSearchBox|CarbonAds|DocAsideCarbonAds|Sponsors)/
```

Comment:
This doesn't seem to be doing anything in current code. For example, the sponsor components are already not in dist/assets/chunks.